curl/Makefile

```
@@ -0,0 +1,40 @@
+# Port Metadata
+PORTNAME =          curl
+PORTVERSION =       7.64.1
+
+MAINTAINER =        Damian Parrino <bucanero@users.noreply.github.com>
+LICENSE =           custom
+SHORT_DESC =        A library for transferring data with URL syntax.
+
+# This port uses CMake.
+PORT_BUILD =        cmake
+
+# Provide SSL support library
+DEPENDENCIES =      zlib polarssl
+
+# What files we need to download, and where from.
+DOWNLOAD_SITE  =    https://curl.haxx.se/download
```

I had to set this in order to retrieve it.
```suggestion
DOWNLOAD_SITE  =    https://curl.se/download
```